### ISSUE TITLE
Could not translate the news: AxiosError: Request failed with status code 504

### ISSUE THREAD

**Issue by SKB-CGN** [AlCalzone/release-script#128]
Hi,
the Command:
`sudo -u pi npm run release -- -p iobroker license`

currently posts the following output:
`check:git git status ok ✔`
`check:package package.json ok ✔`
`check:changelog changelog ok ✔`
`check:iobroker io-package.json ok ✔`
`check:version Bumping version from 0.7.0 to 0.7.1 ✔`
`edit:package updating package.json version from 0.7.0 to 0.7.1`
`edit:changelog Updating changelog in README.md`
`edit:iobroker updating io-package.json version to 0.7.1`
`edit:iobroker updating news in io-package.json`
`edit:iobroker adding new news to io-package.json`
`edit:iobroker [FATAL] Could not translate the news: AxiosError: Request failed with status code 504`

Therefore, the Version is updated in certain files, while io-pckage.json is not update due to the above mentioned error.

The changes were:
chore(deps-dev): bump eslint from 8.19.0 to 8.20.0
chore(deps-dev): bump @types/node from 18.0.1 to 18.6.3
chore(deps-dev): bump @types/sinon from 10.0.12 to 10.0.13

**Comment by SKB-CGN** [comment]
Noted, this error comes, if the elements under WORK IN PROGRESS are too high >6

**Comment by afuerhoff** [comment]
I have the same error today. Elements under WORK IN PROGRESS only two.

edit:iobroker updating news in io-package.json
edit:iobroker adding new news to io-package.json
edit:iobroker [FATAL] Could not translate the news: AxiosError: Request failed with status code 504

**Comment by AlCalzone** [comment]
@Apollon77 is this because of your outage?

**Comment by Apollon77** [comment]
Yes - sorry

**Comment by raintonr** [comment]
I'm seeing this error today. https://translator.iobroker.in/translator works fine in the browser but I get:

```
$ npm run release

> iobroker.hikvision-alarmserver@0.0.5 release
> release-script

git status is good - I can continue...
bumping version 0.0.5 to patch version 0.0.6

updating package.json from 0.0.5 to 0.0.6
updating README.md
updating io-package.json from 0.0.5 to 0.0.6
adding new news to io-package.json...

ERROR: could not translate the news: Error: Request failed with status code 504
```

**Comment by Apollon77** [comment]
Yes, The Internt to the system where the translatioon servcie is hosted is still down ... please be patient

**Comment by afuerhoff** [comment]
Hello, now it's running without errors.

**Comment by Apollon77** [comment]
Yes, internet fixed after 2,5days

**Comment by misanorot** [comment]
> I'm seeing this error today. https://translator.iobroker.in/translator works fine in the browser but I get:
> 
> ```
> $ npm run release
> 
> > iobroker.hikvision-alarmserver@0.0.5 release
> > release-script
> 
> git status is good - I can continue...
> bumping version 0.0.5 to patch version 0.0.6
> 
> updating package.json from 0.0.5 to 0.0.6
> updating README.md
> updating io-package.json from 0.0.5 to 0.0.6
> adding new news to io-package.json...
> 
> ERROR: could not translate the news: Error: Request failed with status code 504
> ```

Same issue here. Yesterday morning it runs perfectly, but since yesterday evening until now, the it doesn't work!

**Comment by Apollon77** [comment]
All fine ... how log the changelog is? Maybe it is too long?

**Comment by misanorot** [comment]
> All fine ... how log the changelog is? Maybe it is too long?

I will check it tomorrow. Thx

**Comment by SKB-CGN** [comment]
Again up with this error, i there are to many elements inside the "WORK IN PROGRESS" list
